luigi/db_task_history.py

```diff
@@ -263,7 +263,9 @@ def _upgrade_schema(engine):
         elif 'mssql' in engine.dialect.name:
             conn.execute('ALTER TABLE task_parameters ALTER COLUMN value TEXT')
         elif 'postgresql' in engine.dialect.name:
-            conn.execute('ALTER TABLE task_parameters ALTER COLUMN value TYPE TEXT')
+            if str([x for x in inspector.get_columns('task_parameters')
+                    if x['name'] == 'value'][0]['type']) != 'TEXT':
+                conn.execute('ALTER TABLE task_parameters ALTER COLUMN value TYPE TEXT')
```

Comment:
what's up with the indentation? Something looks wrong. Maybe mix of spaces and tabs?